android/app/src/main/AndroidManifest.xml

```diff
@@ -17,10 +17,17 @@
         android:launchMode="singleTask"
         android:windowSoftInputMode="adjustResize"
         android:exported="true">
-        <intent-filter>
+        <intent-filter android:autoVerify="true">
             <action android:name="android.intent.action.MAIN" />
             <category android:name="android.intent.category.LAUNCHER" />
         </intent-filter>
+          <intent-filter>
+              <action android:name="android.intent.action.VIEW" />
+              <category android:name="android.intent.category.DEFAULT" />
+              <category android:name="android.intent.category.BROWSABLE" />
+              <data android:scheme="mychat" />
+              <data android:scheme="amiapp" android:host="showtrial" />
```

Comment:
Is `mychat` deliberate here? Also, if we want to support all `amiapp` links, we can skip the host like we did for `mychat`
```
<data android:scheme="amiapp" />
```
However, if you want to keep support for only `amiapp://showtrial`, then no changes required to it

Reply:
Looks perfect then 👍🏻